src/Driver.java

```diff
@@ -64,16 +64,16 @@ public static Date promptDate(String type) {
     }
 
     public static void promptDeleteReservation() {
-    	System.out.println("Enter the a valid integer to delete your reservation");
-        String reservationId = scanner.next();
-        while(!Validator.validateIntegers(reservationId)) {
-            System.out.println("Please enter a valid integer to continue. \n");
-            reservationId = scanner.next();
+    	System.out.println("Enter the reservation id to delete:");
+        int reservationId = scanner.nextInt();
```

Comment:
@ajayreddy-1 We don't need to parsing or validating primitive int type. nextInt() works fine!